test/cache.js

```diff
@@ -28,3 +29,12 @@ test('entries are not cached using buffers from default slab', async function (t
 
   await db.close()
 })
+
+test('maxCacheSize arg', async function (t) {
+  const core = new Hypercore(RAM.reusable())
+  const db = new Hyperbee(core, { maxCacheSize: 10 })
+
+  // Note: tests private props, so not ideal
```

Comment:
Can remove the test if you prefer, since it accesses private props (couldn't find a straightforward way to test this through the public api)